src/Stack/Init.hs

```diff
@@ -281,14 +284,9 @@ renderStackYaml p ignoredPackages dupPackages =
         , "   git: https://github.com/commercialhaskell/stack.git"
         , "   commit: e7b331f14bcffb8367cd58fbfc8b40ec7642100a"
         , "- location: https://github.com/commercialhaskell/stack/commit/e7b331f14bcffb8367cd58fbfc8b40ec7642100a"
-        , "  extra-dep: true"
```

Comment:
Hmm, did this become a default? `extra-dep: true` is probably still good to recommend.  Alternatively, now they can go in extra-deps, maybe it'd be best to switch to the new syntax (see https://www.fpcomplete.com/blog/2017/07/stacks-new-extensible-snapshots)

Reply:
The post claims `extra-dep: true` has caused enough confusion to be worth removing, even at the expense of backwards compatilbility — whether or not it is the default. If this is indeed the case then the earlier the default switches to the new syntax, the better.

Reply:
:+1: